src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -316,6 +316,9 @@
     <ReferenceFromRuntime Include="System.Private.Interop" />
     <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
     <ProjectReference Include="..\..\System.Private.Uri\src\System.Private.Uri.csproj" />
+    <ProjectReference Include="..\..\System.Security.Principal\src\System.Security.Principal.csproj">
+      <Aliases>System_Security_Principal</Aliases>
```

Comment:
Path.Uap.cs will go away because it will come from CoreLib instead, so this does not really matter.

We want to have the behavior to be as close as possible between uap and uapaot. If Path.Uap.cs was not going away, we would want to use Path.Uap.cs for both uap and uapaot builds.